exasol_transformers_extension/udfs/models/base_model_udf.py

```diff
@@ -224,6 +228,24 @@ def get_prediction(self, model_df: pd.DataFrame) -> pd.DataFrame:
         pred_df['error_message'] = None
         return pred_df
 
+    def create_new_span_columns(self, model_df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Maybe as abstract method or a default implementation that does nothing it might be ok

Reply:
we need it in this class in case an error occurs (for example while loading the model). in this case the output still has to have the correct format otherwise the user cont get to the traceback. i will do abstract methods for both functions and move implementation to the udf